django/thunderstore/api/cyberstorm/tests/test_user.py

```diff
@@ -0,0 +1,67 @@
+import json
+
+import pytest
+from django.contrib.auth import get_user_model
+from rest_framework.test import APIClient
+
+from thunderstore.core.types import UserType
+from thunderstore.repository.factories import UserFactory
+
+User = get_user_model()
+
+
+@pytest.mark.django_db
+def test_user_delete__success(
```

Comment:
I've been trying to improve the naming of test cases. Don't know if matters much. But this could be e.g. `test_user_delete__when_deleting_own_account__succeeds`